src/appsco-audit-log-page.html

```diff
@@ -393,6 +430,10 @@
                 }
             },
 
+            _computeSearchAccountsApi: function(companyApi, term) {
+                return companyApi + '/directory/roles?extended=1&limit=50&term=' + term;
```

Comment:
Should compute API only if companyApi and term are not empty.
****
return (companyApi && term) ? 'computed API' : '';
****